### DISCUSSION TITLE
Rename app on marketplace

### DISCUSSION THREAD

**Discussion by richtier** [community/community#27708] [Other Feature Feedback, Questions, & Ideas]
I have a marketplace app https://github.com/marketplace/django-doctor/

but since creating the app the product was renamed to code-review-doctor

I cannot see any way to rename the app and the link. Is this achievable?

Would be less confusing to customers when I call it "code review doctor" but the link says django-doctor

**Comment by EliasAfara** [comment]
Check out [this](https://github.com/orgs/community/discussions/27118#discussioncomment-3254691) answer, I believe it might be helpful in solving your issue.
